src/category_theory/limits/preserves/shapes/pullbacks.lean

```diff
@@ -85,6 +85,27 @@ begin
   apply i,
 end
 
+open walking_cospan
+
+/-- If `F` preserves the pullback of `f, g`, it also preserves the pullback of `g, f`. -/
+def preserves_pullback_symmetry {D : Type*} [category D] (F : C ⥤ D) {X Y Z : C} (f : X ⟶ Z)
```

Comment:
Can you please dualize this for pushouts?

Reply:
I dualized the whole file since I need results from them to dualize this.